daemon/regmaild/regmaild.c

```diff
@@ -270,6 +276,202 @@ regmaildb_update_email(const char * userid, int userid_len, const char * email,
     return ret;
 }
 
+static void
+verifydb_count(const verifydb_req *req, verifydb_count_rep *rep)
+{
+    sqlite3 *Db = g_Db;
+    sqlite3_stmt *Stmt = NULL;
+    int r = SQLITE_OK;
+
+    rep->cb = sizeof(*rep);
+    rep->status = VERIFYDB_ERROR;
+
+    r = sqlite3_prepare(Db, "SELECT userid, generation FROM verifydb "
+            "WHERE vmethod = ? AND vkey = ?;", -1, &Stmt, NULL);
+    if (r != SQLITE_OK)
+        goto end;
+
+    r = sqlite3_bind_int64(Stmt, 1, req->vmethod);
+    if (r != SQLITE_OK)
+        goto end;
+    r = sqlite3_bind_text(Stmt, 2, req->vkey, -1, SQLITE_STATIC);
+    if (r != SQLITE_OK)
+        goto end;
+
+    while (sqlite3_step(Stmt) == SQLITE_ROW) {
+        userec_t u = {};
+        const char *userid = (const char *)sqlite3_column_text(Stmt, 0);
+        int64_t generation = sqlite3_column_int64(Stmt, 1);
+
+        if (userid == NULL)
+            continue;  // Bad record.
```

Comment:
You can get rid of this case by checking for `userid IS NOT NULL` in the `WHERE` clause.